libs/input-mapping/src/Helper/RealDevStorageTagsRewriteHelper.php

```diff
@@ -108,13 +108,15 @@ public function rewriteFileTags(
                 (int) $clientWrapper->getDefaultBranch()->id,
             );
         }
-        /* at this point, nothing has changed - no tags are specified in the configuration */
+
+        /* at this point, nothing has changed - no re-writeable options are specified in the configuration */
+        $logger->info('File input mapping kept unchanged.');
         return new RewrittenInputFileOptions(
-            $fileConfiguration,
+            $fileConfigurationOriginal->getDefinition(),
             $fileConfigurationOriginal->isDevBranch(),
             $fileConfigurationOriginal->getRunId(),
             $fileConfigurationOriginal->getDefinition(),
-            (int) $clientWrapper->getDefaultBranch()->id,
+            (int) $clientWrapper->getBranchId(),
```

Comment:
Nakonec jsem to tady rovnou změnil (a přidal ten log), protože tohle celý se týká jen RealDevStorage, takže to maximálně rozbije nějakej edge-case v SOX projektech a normálních projektů se to vůbec netýká